swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        
+    def get_category(self):
+        return "Item"
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
We don't need to hardcode the word "Item" in this sentence because we have the `get_category` function. 

If you dynamically call `get_category` then `__str__` becomes more flexible and the child classes of `Item` can also use `__str__` instead of repeating code. 

```suggestion
        return f"An object of type {self.get_category} with id {self.id}."
```